Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN R -e 'install.packages( \
   c("V8", "odin", "deSolve", "jsonlite", "remotes"))'
 
 RUN R -e 'library(remotes); install_github(c("mrc-ide/odin.js", \
-"mrc-ide/squire@squire_js"))'
+"mrc-ide/squire@f4c509b"))'
```

Comment:
This looks like it's now just behind master? Do you want to point at a symbolic reference instead?

Reply:
Ah! This is mainline now, no need for the silly refs. Thanks